app/models/recipe.rb

```diff
@@ -4,4 +4,14 @@ class Recipe < ApplicationRecord
 
   validates :name, presence: true
   validates :description, presence: true
+
+  def count_food_recipes
```

Comment:
Would it make sense to rename this to `total_food_items`? This would keep consistency between the method's name and the title of the paragraph where it's being used.